### DISCUSSION TITLE
3.3.0 When is GA?

### DISCUSSION THREAD

**Discussion by luxiao0000** [apache/dubbo#13976] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar feature requirement.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Descriptions

3.3.0 When is GA?

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment, marked as answer]
We plan to release it at June. We are working on testing now.

**Reply by luxiao0000** [reply]
Will it be released in June?
